.babelrc

```diff
@@ -1,5 +1,6 @@
 {
   "presets": ["es2015"],
+  "plugins": ["transform-object-rest-spread"],
```

Comment:
Oh you had wanted this? I reverted it, because it seems like I won’t need it after all :/

Let me know if anyone wants this in anyways and I’ll re-apply the patches.